app/(tabs)/more/profile.tsx

```diff
@@ -133,26 +127,33 @@ const ProfileScreen: React.FC = () => {
                 </TouchableOpacity>
               </Modal>
 
-              <Pressable onPress={handleQrPress}>
-                <View style={styles.qrContainer}>
-                  <SvgQRCode size={130} value={ticketData.hash} />
-                </View>
-              </Pressable>
-
-              <Separator />
               <Profile
                 firstName={ticketData.firstName}
                 lastName={ticketData.lastName}
                 beforeNoon={ticketData.workshopBeforeNoon}
                 participantType={ticketData.ticketType}
-              />
-              <Separator />
+              >
```

Comment:
Voor sommige deelnemers is de naam langer, waardoor de QR code naar onder springt